packages/@aws-cdk/aws-codedeploy/README.md

```diff
@@ -327,3 +292,107 @@ const deploymentGroup = codedeploy.LambdaDeploymentGroup.fromLambdaDeploymentGro
   deploymentGroupName: 'MyExistingDeploymentGroup',
 });
 ```
+
+## Lambda Deployment Configurations
+
+CodeDeploy for Lambda comes with built-in configurations for traffic shifting.
+If you want to specify your own strategy,
+you can do so with the LambdaDeploymentConfig construct,
+letting you specify precisely how fast a new function version is deployed.
+
+```ts
+const config = new codedeploy.LambdaDeploymentConfig(this, 'CustomConfig', {
+  trafficRoutingConfig: new codedeploy.TimeBasedCanaryTrafficRoutingConfig({
+    interval: cdk.Duration.minutes(15),
+    percentage: 5,
+  }),
+});
+
+declare const application: codedeploy.LambdaApplication;
+declare const alias: lambda.Alias;
+const deploymentGroup = new codedeploy.LambdaDeploymentGroup(this, 'BlueGreenDeployment', {
+  application,
+  alias,
+  deploymentConfig: config,
+});
+```
+
+You can specify a custom name for your deployment config, but if you do you will not be able to update the interval/percentage through CDK.
+
+```ts
+const config = new codedeploy.LambdaDeploymentConfig(this, 'CustomConfig', {
+  trafficRoutingConfig: new codedeploy.TimeBasedCanaryTrafficRoutingConfig({
+    interval: cdk.Duration.minutes(15),
+    percentage: 5,
+  }),
+  deploymentConfigName: 'MyDeploymentConfig',
+});
+```
+
+To import an already existing Deployment Config:
+
+```ts
+const deploymentConfig = codedeploy.LambdaDeploymentConfig.fromLambdaDeploymentConfigName(
+  this,
+  'ExistingDeploymentConfiguration',
+  'MyExistingDeploymentConfiguration',
+);
+```
+
+## ECS Applications
```

Comment:
From this README. I can't tell how I would actually deploy an ECS application using these
constructs. Can you add an example showing how you would use these constructs to setup a blue/green
deployment of an actual ECS service?

Reply:
I will make a follow-up PR for the deployment group L2 construct that fills in these details in the README.  I wanted to keep this PR relatively small.
https://github.com/clareliguori/aws-cdk/commit/7d29f934e72057c4606e4f8438565461ec82ccd0